cli/src/semgrep/env.py

```diff
@@ -50,6 +50,13 @@ class Env:
     between multiple invocations.
     """
 
+    fail_open_url: str = field(
+        default=EnvFactory(
+            ["FAIL_OPEN_URL", "SEMGREP_FAIL_OPEN_URL"],
```

Comment:
Do we need the non semgrep prefixed envvar? Also the default url seems ominous :D fail-open and failure

Reply:
No, removing in next commit!